src/app/providers/ContextProviders.tsx

```diff
@@ -11,17 +13,41 @@ import { BoosterProvider } from './NFT/BoosterContext'
 import { MainContainer } from '@/components/MainContainer/MainContainer'
 import { BalancesProvider } from '@/app/user/Balances/context/BalancesContext'
 import { HeroCollapseProvider } from '@/app/user/HeroSection/HeroCollapseContext'
+import { rootstock, rootstockTestnet } from '@reown/appkit/networks'
 
 interface Props {
   children: ReactNode
 }
 
+// Set up metadata
+const metadata = {
+  name: 'Rootstock Collective',
+  description: 'AppKit for the Rootstock Collective',
+  url: process.env.NEXT_PUBLIC_REOWN_METADATA_URL as string, // origin must match your domain & subdomain
```

Comment:
add fallback? seems that not all envs define this

Reply:
Done